src/etc/one-context.d/loc-10-network##rpm.one

```diff
@@ -105,7 +105,11 @@ IPADDR=$IP
 EOT
 
     if [ -n "$GATEWAY" ]; then
-        echo "GATEWAY=$GATEWAY"
+        if [ "$CONFIG_PATH" = "/etc/sysconfig/network" ]; then
+            echo "default $GATEWAY - -" >> /etc/sysconfig/network/routes
```

Comment:
If using `/etc/sysconfig/network/routes`, I wouldn't skip the interface specification in the 4th column ([docs](https://doc.opensuse.org/documentation/leap/reference/html/book.opensuse.reference/cha.basicnet.html#sec.basicnet.manconf.files.routes): *If you leave it empty using a dash (-), it can cause unintended behavior in /etc/sysconfig/network/routes. For more information, see the routes man page.*). Also, the default route needs to be updated if already exists (e.g. when the interface of the existing instance is replaced).

So, I'm not sure if it's not easier and better to work with per-interface routes in `ifroute-ethX` and leave `routes` untouched here for global/user overrides?